<p>This rule detects non final parameters in methods and functions</p>
<h2>Noncompliant Code Example</h2>
<pre>
public void foo(String bar)
</pre>
<h2>Compliant Solution</h2>
<pre>
public void foo(final String bar)
</pre>
